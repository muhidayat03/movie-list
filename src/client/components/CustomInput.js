import styled from 'styled-components';


const CustomInput = styled.input`
    color: white;
    padding: 10px;
    border-radius: 4px;
    z-index: 5;  
    border: 2px solid white;
    box-shadow: none;
    display: inline-block; 
    width: 100%;
    outline: none;
    font-weight: 300;
    font-size: 18px;
    background-color : transparent;
`;

export default CustomInput;