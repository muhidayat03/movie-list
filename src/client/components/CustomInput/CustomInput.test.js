/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import CustomInput from './index';
import toJson from "enzyme-to-json"


 
describe('CustomInput', () => {
  it('should render correctly ', () => {
    const wrapper = shallow(<CustomInput />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
