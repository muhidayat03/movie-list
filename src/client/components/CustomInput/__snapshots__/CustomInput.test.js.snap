// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomInput should render correctly  1`] = `
<input
  className="sc-bdfBwQ jXMrTF"
/>
`;
